core/src/main/java/edu/wpi/grip/core/serialization/ProjectSettingsConverter.java

```diff
@@ -0,0 +1,33 @@
+package edu.wpi.grip.core.serialization;
+
+import com.google.common.eventbus.EventBus;
+import com.thoughtworks.xstream.converters.UnmarshallingContext;
+import com.thoughtworks.xstream.converters.reflection.ReflectionConverter;
+import com.thoughtworks.xstream.io.HierarchicalStreamReader;
+import edu.wpi.grip.core.ProjectSettings;
+import edu.wpi.grip.core.events.ProjectSettingsChangedEvent;
+
+import javax.inject.Inject;
+
+/**
+ * XStream converter for {@link ProjectSettings}.
+ * <p>
+ * Settings are serialized using XStream's built-in reflection serialization. The only catch is that a
+ * {@link ProjectSettingsChangedEvent} must be posted after new settings are loaded.
+ */
+public class ProjectSettingsConverter extends ReflectionConverter {
```

Comment:
# No

I document that we use XStream's reflection serialization already.  Since we're not overriding the bean converter or anything, just not using it, I see no reason to mention it at all in the documentation